tools/travis/deploy.sh

```diff
@@ -0,0 +1,25 @@
+#!/bin/bash
+set -eu
+
+docker_image_prefix="testing"
+dockerhub_image_prefix="openwhisk"
+docker login -u "${DOCKER_USER}" -p "${DOCKER_PASS}"
+
+#capture couchdb setup
+docker commit couchdb "${docker_image_prefix}/couchdb"
+
+#deploy specific openwhisk core images
+for container in "controller"     \
+                 "invoker"        \
+                 "couchdb"        \
```

Comment:
Hi @ddragosd 
I think just taking a snapshot of the couchdb after wipe, its simple and no need to mess with ansible inside the container.
take a look at my latest changes